csrfguard-extensions/csrfguard-jsp-tags/src/main/java/org/owasp/csrfguard/tag/TokenNameTag.java

```diff
@@ -40,14 +40,16 @@
 
 	@Override
 	public int doStartTag() {
-		final String tokenName = CsrfGuard.getInstance().getTokenName();
-
-		try {
-			this.pageContext.getOut().write(tokenName);
-		} catch (final IOException e) {
-			this.pageContext.getServletContext().log(e.getLocalizedMessage(), e);
+		final CsrfGuard csrfGuard = CsrfGuard.getInstance();
+		final String tokenName = csrfGuard.getTokenName();
+
+		if(csrfGuard.isEnabled()) {
```

Comment:
please make sure to follow the same formatting as the rest of the code, and put a space after the `if` keyword.